File: python/circle.py
import math

class Circle:
    def __init__(self):
        self.radius = 0

    def set_radius(self, radius):
        self.radius = radius

    def get_radius(self):
        return self.radius

    def get_diameter(self):
        return 2 * self.radius

    def get_circumference(self):
        return 2 * math.pi * self.radius

    def get_area(self):
        return math.pi * math.pow(self.radius, 2)

File: java/Circle.java
package my.languages;

public class Circle {
  private double radius;
  
  public Circle() {
    radius = 0;
  }
  
  public void setRadius(double r) {
    radius = r;
  }
  
  public double getRadius() {
    return radius;
  }
  
  public double getDiameter() {
      return (2.0 * radius);
  }

  public double getCircumference() {
      return (2.0 * Math.PI * radius);
  }

  public double getArea() {
      return (Math.PI * Math.pow(radius, 2.0));
  }
}

